vermouth/pdb/pdb.py

```diff
@@ -443,6 +459,58 @@ def _do_single_conect(self, conect_record, id2idxs):
                             mol2.nodes[atomidx]['position'])
             mol.add_edge(atomidx0, atomidx, distance=dist)
 
+    def do_seqres(self):
+        """
+        Get the AA sequences of the chains in the molecules
+        """
+        chains = list(list(np.unique(list(mol.nodes[idx]['chain'] for idx in mol)))
+                      for mol in self.molecules)
+
+        properties = {a: [] for a in [x for xs in chains for x in xs]}
+
+        for line in self._seqres:
+            chain = line[11]
+            resnames = line[19:].split()
```

Comment:
As mentioned earlier, I suggest moving this to the method `seqres`. This also makes dealing with malformatted seqres lines easier